src/Persistence/Initialization/Updates/FixChaosMixesPlugInSeason6.cs

```diff
@@ -0,0 +1,422 @@
+            {
+                settings.Money = 0;
+                settings.MoneyPerFinalSuccessPercentage = 200_000;
+                settings.SuccessPercent = 1;
+
+                foreach (var requiredItem in settings.RequiredItems)
+                {
+                    if (requiredItem.FailResult != MixResult.Disappear)
+                    {
+                        // exc item
+                        requiredItem.MinimumAmount = 1;
+                        requiredItem.FailResult = MixResult.ThirdWingsDowngradedRandom;
+                    }
+                    else
+                    {
+                        requiredItem.MaximumAmount = 1;
+                    }
+                }
+
+                settings.ResultItemLuckOptionChance = 5;
+                settings.ResultItemExcellentOptionChance = 0;
+                settings.ResultItemMaxExcOptionCount = 0;
+            }
+        }
+
+        // Cape crafting
+        if (craftings.Single(c => c.Number == 24) is { } capeCrafting)
+        {
+            capeCrafting.ItemCraftingHandlerClassName = typeof(SecondWingsCrafting).FullName!;
+
+            if (capeCrafting.SimpleCraftingSettings is { } settings)
+            {
+                settings.MoneyPerFinalSuccessPercentage = 0;
+
+                foreach (var requiredItem in settings.RequiredItems)
+                {
+                    if (requiredItem.FailResult != MixResult.Disappear)
+                    {
+                        // 1st level wings or exc item
+                        requiredItem.FailResult = MixResult.Disappear;
+
+                        if (requiredItem.NpcPriceDivisor == 4_000_000)
+                        {
+                            // 1st level wings
+                            requiredItem.MinimumItemLevel = 0;
+                        }
+                    }
+                    else
+                    {
+                        // chaos or crest
+                        requiredItem.AddPercentage = 0;
+                        requiredItem.MaximumAmount = 1;
+                    }
+                }
+
+                settings.ResultItemLuckOptionChance = 20;
+                settings.ResultItemExcellentOptionChance = 20;
+                settings.ResultItemMaxExcOptionCount = 1;
+            }
+        }
+
+        // Fruit crafting
+        if (craftings.Single(c => c.Number == 6) is { } fruitCrafting)
+        {
+            fruitCrafting.ItemCraftingHandlerClassName = typeof(SecondWingsCrafting).FullName!;
+
+            if (fruitCrafting.SimpleCraftingSettings is { } settings)
+            {
+                settings.MoneyPerFinalSuccessPercentage = 0;
+
+                foreach (var resultItem in settings.ResultItems)
+                {
+                    resultItem.RandomMaximumLevel = 0;
+                }
+            }
+        }
+
+        // Dark Horse crafting
+        if (craftings.Single(c => c.Number == 13) is { } darkHorseCrafting)
```

Comment:
it's not found, because the dark horse is configured to be crafted in the pet trainer npc.